tests/data/scad/misc/text-metrics-test.scad

```diff
@@ -0,0 +1,25 @@
+use <../../ttf/liberation-2.00.1/LiberationSans-Regular.ttf>
+
+echo("Normal...");
+
+// Force Liberation Sans since it's present on all platforms.
+// Exercise normal operation with default and explicit settings.
+
+// textmetrics()
+echo(textmetrics("hello", font="Liberation Sans"));
+echo(textmetrics("hello", font="Liberation Sans", size=20, direction="rtl",
+    language="en", script="latin", halign="right", valign="center", spacing=2));
```

Comment:
Add tests for empty string and for whitespace only.

Consider adding tests for no text specified (but maybe not, because maybe `text()` should complain if you don't specify text).